changelog.md

```diff
@@ -1,2 +1,11 @@
-#v0.1
+v0.1.1
+------
+
+* Fix a deployment bug by adding `MANIFEST.in`.
+* Add a missing requirement for `django`.
```

Comment:
I can't see the change adding django to the requirements, but FWIW, it was deliberately excluded.